pandas/tests/test_index.py

```diff
@@ -3096,17 +3095,16 @@ def test_join_right(self):
         self.assertIsNone(ridx)
 
         # non-unique
-        """
-        idx = Index([1,1,2,5])
-        idx2 = Index([1,2,5,7,9])
+        idx = Index([1, 1, 2, 5])
+        idx2 = Index([1, 2, 5, 7, 9])
         res, lidx, ridx = idx.join(idx2, how='right', return_indexers=True)
-        eres = idx2
-        elidx = np.array([0, 2, 3, -1, -1])
-        eridx = np.array([0, 1, 2, 3, 4])
+        eres = Index([1, 1, 2, 5, 7, 9])  # 1 is in idx2, so it should be x2
+        elidx = np.array([0, 1, 2, 3, -1, -1])
+        eridx = np.array([0, 0, 1, 2, 3, 4])
         self.assertTrue(res.equals(eres))
         tm.assert_numpy_array_equal(lidx, elidx)
         tm.assert_numpy_array_equal(ridx, eridx)
-
+        """
```

Comment:
I don't know. They are the remnants of some old tests that had been commented because they were not working (at least, it seems like it). I've corrected and adapted the most part, but I have no idea what the rest was intended for. 

I've had a conservative approach, and kept them, but sure I can do a bit of cleaning and remove them. What should I do?